docs/src/modules/components/TopLayoutBlog.js

```diff
@@ -150,7 +171,8 @@ function TopLayoutBlog(props) {
             className={classes.back}
           >
             {/* eslint-disable-next-line material-ui/no-hardcoded-labels */}
-            {'< Back to blog'}
+            <ChevronLeftRoundedIcon fontSize="small" sx={{ mr: 0.5 }} />
+            {'Back to blog'}
```

Comment:
@siriwatknp This one was being displayed as a lint error and I didn't know how to resolve it!

Reply:
Fixed!